specification/cognitiveservices/data-plane/Speech/TextToSpeech/preview/2024-02-01-preview/texttospeech.json

```diff
@@ -2272,11 +2272,11 @@
     "ModelCapability": {
       "description": "Model capability",
       "enum": [
-        "InstanceVoice"
+        "PersonalVoice"
```

Comment:
This is a not a breaking change. Service returns "PersonalVoice" form beginning. It's document issue.